account_payment_sale/views/sale_order_view.xml

```diff
@@ -12,7 +12,6 @@
             <field name="payment_term_id" position="after">
                 <field
                     name="payment_mode_id"
-                    domain="[('payment_type', '=', 'inbound')]"
```

Comment:
Why removing this?

Reply:
because the domain is now defined in the field definition.